ui/src/overview.tsx

```diff
@@ -190,6 +194,39 @@ function ParticipantsGroup(props: { group: tanagra.Group; index: number }) {
     ],
   });
 
+  const fetchGroupCount = useCallback(async () => {
+    const groupForFilter: tanagra.Group = {
+      id: props.group.id,
+      kind: tanagra.GroupKindEnum.Included,
+      criteria: props.group.criteria,
+    };
+
+    const cohortForFilter: tanagra.Cohort = {
+      id: cohort.id,
+      name: cohort.name,
+      underlayName: cohort.underlayName,
+      groups: [groupForFilter],
+    };
+
+    const searchEntityCountsRequest: tanagra.SearchEntityCountsRequest = {
+      entityCounts: {
+        entityVariable: "p",
+        additionalSelectedAttributes: [],
```

Comment:
Can you just omit these instead of passing empty arrays?

Reply:
I checked and they're optional, so I've omitted them.